src/query.ts

```diff
@@ -7,27 +11,36 @@ async function run(): Promise<void> {
   try {
     const query = getInput("query", { required: true });
     const language = getInput("language", { required: true });
-    const nwo = getInput("repository", { required: true });
+    const nwos: string[] = JSON.parse(
+      getInput("repositories", { required: true })
+    );
     const token = getInput("token", { required: true });
     const codeql = getInput("codeql", { required: true });
 
     setSecret(token);
 
-    // 1. Use the GitHub API to download the database using token
-    const dbZip = await downloadDatabase(token, nwo, language);
-    await unbundleDatabase(dbZip);
+    const curDir = cwd();
+    for (const nwo of nwos) {
+      const safeNwo = nwo.replace("/", "#");
```

Comment:
Also - is that certain to be safe? Could use something like [filenamify](https://www.npmjs.com/package/filenamify) otherwise. I don't know what characters we allow - can you have emoji names for example, and can the filesystem handle them?

Reply:
I think the answer is that github doesn't allow any non-filename-safe characters (both linux and windows) in the repository name, or at least I couldn't use any when trying to create a repo just now. That doesn't mean there aren't repos out there with them. Using something like `filenamify` seems sensible to me too. There's also stuff like `com` to worry about, but maybe handling those is not worth it.

Reply:
@robertbrignull on further thought, they didn't allow emoji domain names, until they did ([get an email account with one for lolz](https://mailoji.com/)) - and I think we should not take for granted that naming will remain safe - why bother having to update this in the crazy 🦄  future if we can simply handle it forever with a lib. 🦄 🦄 🦄

Reply:
:+1: `filenamify it is!